opensearch-api/lib/opensearch/api/actions/snapshot/clone.rb

```diff
@@ -33,7 +33,8 @@ module Actions
         # @option arguments [String] :repository A repository name
         # @option arguments [String] :snapshot The name of the snapshot to clone from
         # @option arguments [String] :target_snapshot The name of the cloned snapshot to create
-        # @option arguments [Time] :master_timeout Explicit operation timeout for connection to master node
+        # @option arguments [Time] :master_timeout (DEPRECATED: use cluster_manager_timeout instead) Explicit operation timeout for connection to master node
+        # @option arguments [Time] :cluster_manager_timeout Explicit operation timeout for connection to cluster_manager node
         # @option arguments [Hash] :headers Custom HTTP headers
         # @option arguments [Hash] :body The snapshot clone definition (*Required*)
         #
```

Comment:
Don't forget to update the ParamsRegristry for all affected methods.

Reply:
@nhtruong I've updated them all I believe. Can you take a look?